docs/source/configuration/external_timelock_service_configs/timelock_client_config.rst

```diff
@@ -77,6 +77,12 @@ Optional parameters:
 Runtime Configuration
 ---------------------
 
+.. danger::
+
+   Although we support live-reloading of the server configuration, AtlasDB needs to know at install time that it
+   should talk to TimeLock. Added the block to a running service using embedded timestamp and lock servers is unsafe,
+   as a rolling restart is likely to cause **SEVERE DATA CORRUPTION**.
```

Comment:
I think the important bit to mention here is that you must specify timelock config in both install and runtime. The error message you get otherwise is not overly helpful:
> The TimestampManagementService is currently unavailable, because configuration contains zero servers.

Reply:
@pkoenig10 This is true up to and including 0.74.0, but was changed in 0.75.0 onwards to allow you to specify runtime only (at least, that was the intention of #2850).

@gsheasby Do you think it would be worth mentioning that this is true for older versions? (e.g. in an `info` block)

Reply:
I think the info block would quickly become unnecessary as products move to 0.75.0+, and this part is already quite "blocky".